lib/libstalkerclient/xmltv.c

```diff
@@ -27,6 +27,8 @@
 
 #include "util.h"
 
+#include <kodi/addon-instance/pvr/EPG.h>
```

Comment:
Try this instead, then maybe it compiles.

```
#include <kodi/c-api/addon-instance/pvr/pvr_epg.h>
```

Reply:
My bad I didn't regenerate the depends/kodi from "xmbc@master" branch (was still mapped to "xbmc@Matrix".

The error is strange since generated `EPG.h` is including `#include "../c-api/addon-instance/pvr.h"`, itself including `#include "pvr/pvr_epg.h"`.
So by transitivity `<kodi/addon-instance/pvr/EPG.h>` should embed `<kodi/c-api/addon-instance/pvr/pvr_epg.h>` as well.

Btw the file `GuideManager.h` is compiling as well with : 
https://github.com/kodi-pvr/pvr.stalker/blob/a9b88cc08bb25a15b9c43f6cb07e5b2b97c79e48/src/GuideManager.h#L18, fr the use of https://github.com/kodi-pvr/pvr.stalker/blob/a9b88cc08bb25a15b9c43f6cb07e5b2b97c79e48/src/GuideManager.h#L35

(it also is if I change L18 with `#include <kodi/c-api/addon-instance/pvr/pvr_epg.h>` but I kept it as it was.

Anyway, I directly included `kodi/c-api/addon-instance/pvr/pvr_epg.h` in my latest commit, as suggested. Thanks.